docs/source/search-runs.rst

```diff
@@ -93,139 +93,183 @@ Then paste the following code to a local pythnon file, e.g., ``~/create_testing_
           mlflow.log_input(dataset, context=dataset_context[i])
 
 
-The code above creates 10 MLflow runs with different metrics, params, tags and dataset information. After execution,
-you should find all these runs under the experiment "/search-run-guide", as shown by the screenshot:
+The code above creates 10 MLflow runs with different metrics, params, tags and dataset information. 
+After successful execution, if you return to the MLflow UI in your browser, you should find all 
+these runs under the experiment "search-run-guide", as shown by the screenshot:
 
 .. figure:: _static/images/search-runs/created_mlflow_runs.png
    :alt: testing runs
    :width: 90%
    :align: center
 
+Now in production systems, it's common to have thousands or even hundreds of thousands of mlflow 
+runs. In such cases, it's important to be able to filter and search for runs based on specific criteria.
 
 .. _search-runs-syntax:
 
-Search Query Syntax
-^^^^^^^^^^^^^^^^^^^
+Search Query Example 
+^^^^^^^^^^^^^^^^^^^^
 
-You need to write search queries to fetch your MLflow runs. The query must be a condition or a few conditions
-concatenated by ``and``. At a high level, the syntax is similar to SQL WHERE clause without supporting ``or`` keyword.
-For example, below is a query to search for all runs that have logged a ``loss`` metric smaller than ``1`` to
-MLflow.
+To filter your MLflow runs, you need to write **search queries**, which are SQL-like conditions. 
 
-.. code-block:: sql
+To showcase this functionality, let's reopen the MLflow UI, let's explore some example fields that you can use to filter your runs in the MLflow UI.
+
+.. figure:: _static/images/search-runs/before_search_query.png
+   :alt: search query
+   :width: 90%
+   :align: center
```

Comment:
Can we use code example instead a screenshot? Screenshots can be outdated easily.

Reply:
+1